package.json

```diff
@@ -1,7 +1,11 @@
 {
-  "name": "purescript-react",
-  "files": [],
-  "peerDependencies": {
-    "react": "^16.3.0"
+  "private": true,
+  "scripts": {
+    "build": "spago build --purs-args '--censor-lib --strict'",
```

Comment:
In a subsequent PR we can enable eslint via `eslint src && spago build ...` here.